public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -97,131 +96,134 @@ <h5 class="col-xs-12 box-title">
-                            Source concepts are the way the data was originally represented in a patient’s record.
-                            The vocabulary code listed here is a standard vocabulary code for the medical information.
-                            Standard codes are ways of representing data that is recorded many different ways in original patient records but essentially means the same thing.
-                        </div>
-                    </clr-tooltip-content>
-                </clr-tooltip>
+            <clr-tooltip>
+              <clr-icon clrTooltipTrigger shape="info-standard" class="info-icon"></clr-icon>
+                <clr-tooltip-content clrPosition=left
+                             clrSize="lg" *clrIfOpen>
+                  <div class="tooltip-label">
+                    Individual health records often contain medical information that means
+                    the same thing but may be recorded in many different ways.
+                    Source concepts are the way the data was originally represented in a patient’s record.
+                    The vocabulary code listed here is a standard vocabulary code for the medical information.
+                    Standard codes are ways of representing data that is recorded many different ways in original patient records but essentially means the same thing.
+                  </div>
+                </clr-tooltip-content>
+            </clr-tooltip>
           </div>
           <div class="col-md-1 grid-header flex-xs-middle info-text"> View graphs
           </div>
         </div>
         <div class="row expandable-grid-row"
              *ngFor="let r of standardConcepts.concat(items), let i = index"
              [ngClass]="{'expanded-grid-row': r.expanded}" >
-            <div class="col-md flex-xs-middle" *ngIf="searchResult.matchType === 'CONCEPT_CODE' && r.standardConcept !='S';else code_match_standard">
-                <div class="col-md flex-xs-middle code-tooltip" innerHtml="<b>Source</b> {{r.conceptName | highlightSearch : prevSearchText}}
-            -- {{r.conceptId}}">
-                </div>
-                <clr-tooltip>
-                    <clr-icon clrTooltipTrigger shape="info-standard" class="info-icon"></clr-icon>
-                    <clr-tooltip-content clrPosition=bottom-right
-                                         clrSize="lg" *clrIfOpen>
-                        <div class="tooltip-label">{{conceptCodeHelpText}}</div>
-                    </clr-tooltip-content>
-                </clr-tooltip>
+          <div class="col-md flex-xs-middle" *ngIf="searchResult.matchType === 'CONCEPT_CODE' && r.standardConcept !='S';else code_match_standard">
+            <div class="col-md flex-xs-middle code-tooltip">
+              <app-html-wrapper [isSource]="true" [markedData]="r.conceptName | highlightSearch : prevSearchText" [conceptId]="r.conceptId"></app-html-wrapper>
```

Comment:
I think this unfortunately still has the same issue. The problem is with this `highlightSearch` pipe. The component I think this is the right idea though. If you could have a highlight-search component I believe you could solve the issue there. Here's a rough outline:

```
HighlightSearchComponent {
  this.chunks = this.text.split(this.searchTerm);
}

// html
<span *ngFor="let chunk of chunks" [class.highlight]="chunk === searchTerm">
  {{chunk}}
</span>
```